### ISSUE TITLE
Use alternative "base url" instead of only the schedule url

### ISSUE THREAD

**Issue by RealRancor** [tuxmobil/CampFahrplan#93]
Hi,

i think the alternative url is a nice feature for other events providing a fahrplan. Currently i'm using the Schedule of https://mrmcd.net/2015/fahrplan/ in the app which works great.

The only thing i've noticed is, that the "Event online" link of an event is pointing to:

https://events.ccc.de/camp/2015/Fahrplan/events/7077.html

instead of:

https://mrmcd.net/2015/fahrplan/events/7077.html

Maybe the alternative URL could be changed so that it is possible to insert an base url like:

https://mrmcd.net/2015/fahrplan/

which is then used for generating all other URLs?

**Comment by johnjohndoe** [comment]
I could reproduce that the "Event online" link is incorrect when I enter an *alternative URL* and select refresh right after. Just to confirm: is this what you did?

**Comment by RealRancor** [comment]
Hi,

yes this is what i did. To reproduce this:

1. Open app
2. Choose settings
3. Choose "alternative URL"
4. Insert https://mrmcd.net/2015/fahrplan/schedule.xml
5. Close settings
6. Refresh

The new schedule of MRMCD 2015 is working without any problems. Only the "Event online" link is pointing to:

https://events.ccc.de/camp/2015/Fahrplan/events/7077.html

instead of

https://mrmcd.net/2015/fahrplan/events/7077.html

**Comment by johnjohndoe** [comment]
I guess you used the [CampFahrplan app](https://play.google.com/store/apps/details?id=nerd.tuxmobil.fahrplan.camp) to test this. I used the [MRMCD15 app](https://play.google.com/store/apps/details?id=info.metadude.android.mrmcd.schedule) and entered the Camp schedule URL to reproduce the behavior.

Yes, you are right. Originally the function was implemented to being able to point to alternative server when the main server is not reachable. This happened at 30c3 a lot.

**Comment by RealRancor** [comment]
> I guess you used the CampFahrplan app to test this.

Yes, thats why i had posted that in here. :) I don't have google apps installed so i don't use the MRMCD app directly but the CampFahrplan from F-droid.

**Comment by johnjohndoe** [comment]
Thank you. You can build the MRMCD15 yourself from [this fork/branch](https://github.com/johnjohndoe/CampFahrplan/tree/mrmcd-2015).

**Comment by RealRancor** [comment]
> You can build the MRMCD15 app yourself from this fork/branch.

Sure i could build the app myself. But the reason of this feature request is that the existing app could be used with an alternative URL so its not needed to rebuild a completely new app.

**Comment by johnjohndoe** [comment]
True. Please mind that the source code of MRMCD15 is not identical to the source code of CCCamp15.
